src/main/kotlin/org/rust/cargo/runconfig/filters/RegexpFileLinkFilter.kt

```diff
@@ -65,9 +77,64 @@ open class RegexpFileLinkFilter(
         }
     }
 
-    private fun createOpenFileHyperlink(fileName: String, line: Int, column: Int): HyperlinkInfo? {
-        val platformNeutralName = fileName.replace(File.separatorChar, '/')
-        val file = cargoProjectDirectory.findFileByMaybeRelativePath(platformNeutralName) ?: return null
-        return OpenFileHyperlinkInfo(project, file, line, column)
+    private fun resolveFilePath(fileName: String): ResolvedPath? {
+        val path = fileName.replace(File.separatorChar, '/')
+        val file = cargoProjectDirectory.findFileByRelativePath(path)
+        if (file != null) return ResolvedPath.Project(file)
+
+        val stdlibFile = resolveStdlibFile(fileName)
+        if (stdlibFile != null) return stdlibFile
+
+        val cargoFile = resolveCargoFile(fileName)
+        if (cargoFile != null) return cargoFile
+
+        // try to resolve absolute path
+        return cargoProjectDirectory.fileSystem.findFileByPath(path)?.let { ResolvedPath.Crate(it) }
```

Comment:
Maybe it's better to introduce something like `Common` or `Unknown` because don't know where the path lead (at least currently)